synapse/lib/iq.py

```diff
@@ -321,11 +375,21 @@ def getRamCore(self):
             self.addTstForms(core)
             try:
                 yield core
-            except:  # pragma: no cover
-                raise
             finally:
                 core.fini()
 
+    @contextlib.contextmanager
+    def getDirCore(self):
+        '''
+        Context manager to make a dir:/// cortex which has test models
+
+        Yields:
+            s_cores_common.Cortex: Dir backed Cortex
+        '''
+        with self.getTestDir() as dirn:
+            with s_cortex.fromdir(dirn) as core:
+                yield core
```

Comment:
Will need to call ``self.addTstForms(core)`` to make sure it has the test models the docstring says it has loaded in it.

Reply:
modifying the doc string.